.github/workflows/build-test.yml

```diff
@@ -227,6 +227,11 @@ jobs:
           docker-compose up --exit-code-from gremlin-go-integration-tests
           docker-compose down
 
+      - name: Upload to Codecov
```

Comment:
was there ever any investigation into whether ASF infra involvement was required to make this work?